packages/esm-sms-app/src/providers/providers-overview/providers-overview.component.tsx

```diff
@@ -0,0 +1,307 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { CardHeader, ErrorState } from '@openmrs/esm-patient-common-lib';
+import { useProviderConfigurations } from '../../hooks/useProviderConfigurations';
+import { useConfigurationsSorting } from '../utils';
+import {
+  Button,
+  InlineLoading,
+  DataTable,
+  DataTableSkeleton,
+  Pagination,
+  TableContainer,
+  Table,
+  TableHead,
+  TableHeader,
+  TableRow,
+  TableBody,
+  TableCell,
+  TableExpandHeader,
+  Tile,
+} from '@carbon/react';
+import { Add } from '@carbon/react/icons';
+import {
+  usePagination,
+  useLayoutType,
+  isDesktop as isDesktopLayout,
+  showModal,
+  launchWorkspace,
+} from '@openmrs/esm-framework';
+import { ConfigurationsActionMenu } from './providers-action-menu.component';
+import { TableExpandRow } from '@carbon/react';
+import { TableExpandedRow } from '@carbon/react';
+import { EmptyState } from '../empty-state/empty-state.component';
+import { Tag } from '@carbon/react';
+import styles from './providers-overview.scss';
+import classNames from 'classnames';
+
+const ProvidersListTable = () => {
+  const { t } = useTranslation();
+  const layout = useLayoutType();
+  const isDesktop = isDesktopLayout(layout);
+  const isTablet = !isDesktop;
+  const pageSizes = [10, 20, 30, 40, 50];
+  const [pageSize, setPageSize] = useState();
+  const { providerConfigurations, isLoadingConfigs, error, isValidatingConfigs } = useProviderConfigurations();
+
+  const launchAddProviderConfigForm = useCallback(
+    () =>
+      launchWorkspace('add-provider-config-form', {
+        workspaceTitle: t('addProviderConfigFormWorkspaceTitle', 'Add Provider'),
```

Comment:
I think this should be the default title. See [here](https://github.com/openmrs/openmrs-esm-patient-chart/blob/503512840f89c232f70e1b8f2cec2eeba87f5185/packages/esm-patient-tests-app/src/routes.json#L51) and [here](https://github.com/openmrs/openmrs-esm-patient-chart/blob/503512840f89c232f70e1b8f2cec2eeba87f5185/packages/esm-patient-tests-app/src/index.ts#L51). Then you can remove the `workspaceTitle` parameter.